app/views/user_mailer/change_email.text.erb

```diff
@@ -1,3 +1,3 @@
 <% content_for :message do %>
-<%= t('.part1', login: @user.login, email: @user.email) %>
+<%= t('.changed.html', login: @user.login, email: @user.email) %>
```

Comment:
Hmm. It _probably_ wouldn't hurt anything for now to use the HTML safe translation in both places, but it'll likely come back to bite us i the future, so could you change this to `.changed.text` and update `en.yml` to include the same text for this key, i.e.,

```
changed:
  html: "%{login}, the email associated with your account has been changed to %{email}"
  text: "%{login}, the email associated with your account has been changed to %{email}"
```

(I don't see any way to easily duplicate the translation for all locales using i18n-tasks, but if you do, that would be even better.)